Dbscan

Ju2t-us opened the pull request:
Add in DBSCAN first implementation (will continue being developed)

Commit:
First DBSCAN implementation

Seems slightly inaccurate: mean age for all clusters is ~54 years

Ju2t-us committed:
Delete ad_details.csv

Ju2t-us committed:
Delete ad_events.csv

Ju2t-us committed:
Delete README.md

Ju2t-us committed:
Delete customer_profiles.csv

Commit:
Update DBSCAN

More condensed, still seems inaccurate though

Commit:
Merge branch 'DBSCAN' of https://github.com/Ju2t-us/RFP into DBSCAN

TestForSchool commented:
Looks Good! When your done I'll start on making the ad.

TestForSchool reviewed (approved):
Looks Good! keep me posted for when your done so I can start making an Ad. Can you please add more print statements so I can follow your process better? Just like a quick statement to tell me what part of the code is running or complete. Thank you!!
-Nico

Commit:
Update DBSCAN

changed DBSCAN implementation so that the print statements were in different cells and easily viewable (per Nico's request)
